packages/block-library/src/navigation-link/block.json

```diff
@@ -8,10 +8,6 @@
 		"label": {
 			"type": "string"
 		},
-		"nofollow": {
```

Comment:
Removing an attribute will cause any existing block markup that's out in the wild to no longer load. We'll need to use the `deprecated` attribute.

https://developer.wordpress.org/block-editor/developers/block-api/block-deprecation/